### PR TITLE
create main-2.x snapshot that contains #19330

### PR DESCRIPTION
**Pull request by paulbrauner-da**
We need this asap for the repo dance.

### CODE REVIEW

File: sdk/LATEST [hunk 1]
```diff
@@ -1,6 +1,6 @@
 a0d93a12df75c0dfef4d31b0dc0e35f318036cb3 3.1.0-adhoc.20240527.13087.0.va0d93a12 SPLIT_RELEASE
 def6b7987a28dde0d363878f58a396eec47f1a5a 3.0.0-snapshot.20240327.12947.0.vdef6b798 SPLIT_RELEASE
-ad262e6df5bde5ba1d83f8c5bbf13c4916a859fc 2.9.0-snapshot.20240603.12818.0.vad262e6d SPLIT_RELEASE
+98b65512accdcdecb5fed140542ff228e552dfef 2.9.0-snapshot.20240603.12820.0.v98b65512 SPLIT_RELEASE
```
Comment (tudor-da) [inline sdk/LATEST:3]: maybe you meant a more recent commit?

Comment (cocreature) [inline sdk/LATEST:3]: are you intentionally releasing from an older commit on main-2.x instead of the latest one that contains your upgrade fixes?

Comment (paulbrauner-da) [reply sdk/LATEST]: yes, thanks!

Comment (paulbrauner-da) [reply sdk/LATEST]: no this wasn't intentional, thanks for spotting!

**Review by cocreature — APPROVED** [review]
thanks!

**Review by tudor-da — APPROVED** [review]
lgtm

**Comment by paulbrauner-da** [comment]
Death by flake, obsolete now. Thank you for all the approvals.

**Pull request closed without merging**